SODA.Utilities/ExcelOleDbHelper.cs

```diff
@@ -9,13 +9,15 @@ namespace SODA.Utilities
     /// <summary>
     /// A helper class for working with Excel over OleDb.
     /// </summary>
+    [Obsolete("Usage of OleDb will be removed in v0.6.0. Use the ExcelDataReaderHelper class instead.")]
     public class ExcelOleDbHelper
     {
-        /// <summary>
+        /// <deprecated type="deprecate">
         /// Creates an OleDbConnection to a specified Excel file.
-        /// </summary>
+        /// </deprecated>
         /// <param name="excelFileName">The path to a readable Excel (.xls or .xlsx) file.</param>
         /// <returns>An OleDbConnection to the specified Excel file.</returns>
+        [Obsolete("This method will be removed in v0.6.0. Use ExcelDataReaderHelper::MakeExcelReader() instead.")]
```

Comment:
Since `MakeExcelReader` was removed in favor of a simpler method for just getting the data, this comment should change. Just remove the *Use ExcelDataReaderHelper...* part